pydicom/_private_dict.py

```diff
@@ -3613,7 +3613,7 @@
         '0019xx40': ('DS', '1', 'Unknown', ''),  # noqa
         '0019xx41': ('DS', '1', 'Unknown', ''),  # noqa
         '0019xx42': ('IS', '1', 'Unknown', ''),  # noqa
-        '0019xx43': ('IS', '1', 'Unknown', ''),  # noqa
+        '0019xx43': ('DS', '1', 'Unknown', ''),  # noqa
```

Comment:
Thanks for that!
We actually generate the private dict from GDCM, so two things:
- before making this change, run `generate_private_dict.py` (under `source`) to update to the latest version 
- probably make a PR in GDCM against their [private dict](https://github.com/malaterre/GDCM/blob/master/Source/DataDictionary/privatedicts.xml)

Also, just to be sure: you are sure that the change is correct (from the DICOM conformance statement or from knowing the Hologic format) as opposed to just wrong in some instances, or changed in some version), right?

Reply:
I don't know the history of the field, whether it used to be IS and was changed to DS or if it has always been DS.

Some of the DICOMs I'm working with have "integer" values for this field (such as b"1.0") but this appears to be a coincidence. Many have values such as:
`(0019, 1043) DS: '1.58'`
`(0019, 1043) DS: '1.93'`
`(0019, 1043) DS: '1.34'`

Without the proposed change in place, a TypeError is raised when a value such as 1.58 is interpreted according to IS. I suppose the other way to handle this is to allow the user to optionally ignore these kinds of TypeErrors.

I opened a corresponding PR on GDCM: https://github.com/malaterre/GDCM/pull/133

Reply:
Thanks, great! Can you please update the private dict as mentioned, and also add an entry in the release notes (`v2.3.0.rst`)?

Reply:
(the update of the private dict is not strictly related to your change, of course, but I just want to ensure that we won't do it after your change and revert that change inadvertedly)

Reply:
No problem. Done.